data/json/monsters/mi-go.json

```diff
@@ -0,0 +1,201 @@
+[
+  {
+    "id": "mon_mi_go",
+    "type": "MONSTER",
+    "name": "mi-go",
+    "description": "This is an alien creature of uncertain origin.  Its shapeless pink body bears numerous sets of paired appendages of unknown function, and a pair of ribbed, membranous wings which seem to be quite useless.  Its odd, vaguely pyramid-shaped head bristles with numerous wavering antennae, and simply gazing upon the unnatural beast fills you with primordial dread.",
+    "default_faction": "mi-go",
+    "bodytype": "migo",
+    "species": [ "NETHER" ],
+    "volume": "92500 ml",
+    "weight": 120000,
+    "hp": 210,
+    "speed": 120,
+    "material": [ "flesh" ],
+    "symbol": "&",
+    "color": "pink",
+    "aggression": 20,
+    "morale": 30,
+    "melee_skill": 7,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 6,
+    "dodge": 4,
+    "armor_bash": 4,
+    "armor_cut": 12,
+    "vision_day": 50,
+    "harvest": "zombie_meatslug",
+    "path_settings": { "max_dist": 50 },
+    "special_attacks": [
+      [ "PARROT", 0 ],
+      { "id": "scratch", "damage_max_instance": [ { "damage_type": "cut", "amount": 23, "armor_multiplier": 0.8 } ] }
+    ],
+    "death_function": [ "NORMAL" ],
+    "flags": [
+      "SEES",
+      "SMELLS",
+      "HEARS",
+      "WARM",
+      "BASHES",
+      "POISON",
+      "NO_BREATHE",
+      "ARTHROPOD_BLOOD",
+      "PATH_AVOID_DANGER_2",
+      "PRIORITIZE_TARGETS"
+    ]
+  },
+  {
+    "id": "mon_mi_go_surgeon",
+    "type": "MONSTER",
+    "name": "mi-go surgeon",
+    "description": "This mi-go has a slender body with snaking carapace along it, and even more paired clawed appendages.  Its claws are smaller and more delicate looking, but that does not make them less unnerving.",
```

Comment:
Maybe explicitly say `more [..] than a regular mi-go`, I'm not sure it's clear enought what it's compared to for someone that would encounter this one first.

Reply:
It shouldn't be possible to encounter this first, it's a rare enemy deep inside a mi-go encampment.

Reply:
It's probably a good idea to generally make descriptions not related to one another. Something like "Its claws, while quite small and delicate in appearance, still seem dextrous and dangerous enough to make one want to avoid them." would work as well without referencing anything.